model-optimizer/mo/front_ng/frontendmanager_wrapper.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python3
+
+# Copyright (C) 2018-2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import os
+import sys
+
+
+def create_fem():
+    fem = None
+    try:
+        from ngraph.frontend import FrontEndManager  # pylint: disable=no-name-in-module,import-error
+        fem = FrontEndManager()
+    except Exception:
+        print("nGraph FrontEndManager is not initialized")
+        pass
+    return fem
+
+
+if __name__ == "__main__":
+    if not create_fem():
+        exit(1)
```

Comment:
I doubt that we need this code. Do we expect that someone will run this file as a script? I don't think so

Reply:
Since 'auto IE detect' is implemented in master, this wrapper is not needed anymore, removed it.